extension/src/components/Graph.tsx

```diff
@@ -326,14 +404,42 @@ export function GraphVisualization() {
 					</g>
 				</svg>
 
-				{/* Reset view button */}
-				<button
-					className="graph-reset-button"
-					onClick={resetView}
-					title="Reset view"
-				>
-					⟲ Reset View
-				</button>
+				{/* Control buttons */}
+				<div className="graph-controls">
+					<button
+						className="graph-reset-button"
+						onClick={resetView}
+						title="Reset view"
+					>
+						⟲ Reset View
+					</button>
+
+					<div ref={exportMenuRef} className="graph-export-container">
+						<button
+							className="graph-export-button"
+							onClick={toggleExportMenu}
+							title="Export graph"
+						>
+							↓ Export
+						</button>
+						{showExportMenu.value && (
+							<div className="graph-export-menu">
+								<button
+									className="graph-export-menu-item"
+									onClick={handleExportMermaid}
+								>
+									Mermaid (clipboard)
```

Comment:
Rather than displaying (clipboard) can we just do a popup that it got copied to their clipboard?